base/logger_external.go

```diff
@@ -184,74 +185,19 @@ func (CBGoUtilsLogger) Logf(level logging.Level, fmt string, args ...interface{}
-}
-
-func (CBGoUtilsLogger) Fatalp(msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Fatalp not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Severem(msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Severem not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Severep(msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Severep not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Errorm(msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Errorm not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Errorp(msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Errorp not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Warnm(msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Warnm not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Warnp(msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Warnp not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Infom(msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Infom not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Infop(msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Infop not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Requestm(rlevel logging.Level, msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Requestm not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Requestp(rlevel logging.Level, msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Requestp not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Tracem(msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Tracem not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Tracep(msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Tracep not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Debugm(msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Debugm not implemented! " + msg)
-}
-
-func (CBGoUtilsLogger) Debugp(msg string, kv ...logging.Pair) {
-	Warnf("CBGoUtilsLogger: Debugp not implemented! " + msg)
-}
 
-func (CBGoUtilsLogger) Logm(level logging.Level, msg string, kv logging.Map) {
-	Warnf("CBGoUtilsLogger: Logm not implemented! " + msg)
+func (l CBGoUtilsLogger) Loga(level logging.Level, f func() string) { l.warnNotImplemented("Loga", f) }
+func (l CBGoUtilsLogger) Debuga(f func() string)                    { l.warnNotImplemented("Debuga", f) }
+func (l CBGoUtilsLogger) Tracea(f func() string)                    { l.warnNotImplemented("Tracea", f) }
+func (l CBGoUtilsLogger) Requesta(rlevel logging.Level, f func() string) {
+	l.warnNotImplemented("Requesta", f)
 }
+func (l CBGoUtilsLogger) Infoa(f func() string)   { l.warnNotImplemented("Infoa", f) }
+func (l CBGoUtilsLogger) Warna(f func() string)   { l.warnNotImplemented("Warna", f) }
+func (l CBGoUtilsLogger) Errora(f func() string)  { l.warnNotImplemented("Errora", f) }
+func (l CBGoUtilsLogger) Severea(f func() string) { l.warnNotImplemented("Severea", f) }
+func (l CBGoUtilsLogger) Fatala(f func() string)  { l.warnNotImplemented("Fatala", f) }
```

Comment:
Many of the removed functions don't seem to have been refactored such as `Infop`, meaning the `Logger` interface would not be implemented.

Reply:
What do you mean by refactored here? This is uptaking a `couchbase/goutils` interface change which removed functions like `Infop` that we're required to update to because of modules.

https://github.com/couchbase/goutils/commit/e33d3ffb5401c6e3c7577bf313e866ba12c2ca2a

Reply:
Whoops was looking at the currently used version.